src/trade.js

```diff
@@ -96,9 +96,9 @@ class Trade extends Exchange.Trade {
         // This log only happens if .set() is called after .debug is set.
         console.info('Trade ' + this.id + ' from Coinify API');
       }
-      this._createdAt = new Date(obj.createTime);
-      this._updatedAt = new Date(obj.updateTime);
-      this._quoteExpireTime = new Date(obj.quoteExpireTime);
+      this._createdAt = new Date(obj.createTime).getTime();
+      this._updatedAt = new Date(obj.updateTime).getTime();
+      this._expiresAt = obj.quoteExpireTime ? new Date(obj.quoteExpireTime).getTime() : new Date().getTime() - 1;
```

Comment:
What do you mean by "the trade was expired"? You mean that the quote was expired, rather than the trade state being `expired`?